util/install-packages.sh

```diff
@@ -2,17 +2,6 @@
 
 set -e
```

Comment:
Do we need `install-packages.sh` now? It is referenced in `QUICK_START.md`, but I suppose people can just `npm i` from the root now instead.
Maybe we can just drop in `npm ci` into the build phase of the pipeline too and remove this altogether.

Reply:
We do not. I'm inclined to remove some of these simpler scripts which can now be just standalone commands. I'll open up a separate PR regarding this and a general script cleanup.